mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -18,14 +18,16 @@
     <!-- No Open Tabs Message Description -->
     <string name="no_open_tabs_description">Your open tabs will be shown here.</string>
 
+    <!-- About content. The first parameter is the name of the application. (For example: Fenix) -->
+    <string name="about_content">%1$s is produced by Mozilla.</string>
```

Comment:
@ekager or @boek 
Can you help me understand if this should land with Mozilla hard-coded or if it should be a placeable?

Reply:
We don't sub "Mozilla" anywhere else 🤔  For example preferences_usage_data_description, preference_experiments_summary, etc. 
I think it should be hardcoded.

Reply:
Thanks, let's keep it this hardcoded for now. 

It might be worth raising the question for the future though:
- This impacts redistribution. Is someone decides to fork the code and redistribute, they can change the name but not the vendor. To be clear, I have no idea how hard it would be to change the name either, I only have some experience with Firefox desktop code :)
- Crash Reporter and Telemetry are an edge case, we probably have those hardcoded in Firefox too. Even if someone decides to redistribute Fenix, they will probably not redirect those services to a different server, they either keep them as is (and send data to Mozilla) or disable them.

Again, worth some more thought, but not a blocker.